src/ml-search.service.js

```diff
@@ -629,9 +629,9 @@
      */
     getCombinedQuery: function getCombinedQuery(includeOptions) {
       var d = $q.defer(),
-          combined = {
-            search: { query: this.getQuery() }
-          };
+        combined = {
+          search: { query: this.getQuery() }
+        };
 
```

Comment:
Auch, sounds like jshint does no longer check indentation since 2.5.0: http://stackoverflow.com/questions/23097316/how-can-i-make-jshint-indent-options-work

Which means my TextMate plugin is using something older? Anyhow, I like the indent hints very much actually.

Maybe start using jslint as well? :(